### DISCUSSION TITLE
I can't register for the student developer pack

### DISCUSSION THREAD

**Discussion by sa1nt9** [community/community#75292] [GitHub Education]
### Hi everyone,

I am receiving an application for a github student developer pack, it says:
"Please correct your GitHub billing information with your last name exactly as it appears in your academic affiliation document. If you have only a single legal name, enter it in both the first and last name fields.
Please correct your GitHub billing information with your first name exactly as it appears in your academic affiliation document. Do not use a variation of your name or a nickname."
No matter how I send or take a photo of the certificate of my studies at school, this error appears, I wrote billing information both in my native language and in English, it gives the same error

**Comment by davixmns** [comment]
Here's a suggestion to resolve the issue:

Check Academic Affiliation Document:
Make sure to carefully review your academic affiliation document to confirm the exact spelling of your first and last name. Pay attention to any special characters or formatting.

Enter Legal Name:
Input your legal name exactly as it appears on the document. If you have a single legal name, enter it in both the first and last name fields.

Avoid Variations or Nicknames:
Ensure that you are not using any variations, abbreviations, or nicknames for your name. Enter it exactly as it is on your academic affiliation document.

Language Consistency:
If you've entered your billing information in both your native language and English, make sure that there is consistency between the two versions. Stick to the exact wording from your academic document.

Contact GitHub Support:
If the issue persists, consider reaching out to GitHub support. They can provide specific assistance and guidance on resolving such billing information errors.

**Comment by MohamedNourDerbeli** [comment]
Here are some steps you can take to troubleshoot and resolve the issue:

Check Your Academic Affiliation Document:

Review your academic affiliation document to ensure that the first and last name matches exactly what you're entering on GitHub. Pay attention to any spaces, special characters, or formatting.

Use Your Legal Name:

Make sure you're using your legal name as it appears on your academic document. Avoid using variations, nicknames, or any other name that might differ from your official records.

Enter Single Legal Name:

If you have only a single legal name, enter it in both the first and last name fields on the GitHub billing information form.

**Comment by ppvan** [comment]
Not sure how it can be wrong. My fullname is "Phạm Văn Phúc" in Vietnamese, and also in the document
But to be exact, my first name must be "Phúc" part and last name "Phạm Văn", kinda the opposite with foreign names. So how can I get my billing info right?

**Reply by hongnhu2005** [reply]
Bác fix được lỗi này chưa bác ơi

**Reply by KlausJackson** [reply]
Just write the name on your billing info EXACTLY like your name on your birth certificate.

**Comment by ghost** [comment]
While filling billing information for Github student developer pack , we have to put home address or college address in the form ???

**Comment by tapantabhanja** [comment]
I am a PhD Student at the Technical University of Kaiserslautern in Germany. I wanted to use the Student Developer Pack. However, during verification of my Student Status, I was rejected on the first try because my Billing Information and 2-Factor Authentication was not set up. Note that, I uploaded my proof of student status. After filling up the required billing information and setting up the 2-Factor Authentication, my application is still not being processed. The error message says that the application can't be reviewed because the uploaded document to prove my student status has already been used. 

I can't change this document. It is an Immatriculation Confimation (in German "Immatrikulationsbescheinigung") provided by the university every semester as a proof of enrolment as a student.
  
What do I do now?

**Reply by samus-aran** [reply]
Hi @tapantabhanja 👋 

We would suggest take another photo of the evidence as we do prefer a live photo captured during the application. Let us know how you get on.

**Comment by samus-aran** [comment]
Hi @Grolland-creator 

I suspect that between your english translation in the billing info and what is being translated by the application process is different. If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍

**Comment by hopl248i** [comment]
My full name is "PHAN LÊ HỒ". I entered "PHAN" as the last name and "LÊ HỒ" as the first name in billing, and it worked.
![image](https://github.com/user-attachments/assets/b93ad5ad-ff8d-41df-a734-cab5cda76987)

**Reply by cuonghuynh2403** [reply]
là phải viết ngược lại hở bác, em có acc cá nhân chưa đăng kí thông tin nhưng add sinh viên được, nhưng tài khoản sinh viên thì làm đủ kiểu vẫn lỗi

**Reply by holedev** [reply]
Đúng rồi bác ơi, định dạng bên đó chắc là lastname + firstname, bác cứ để họ ở lastname còn lại bỏ vô firstname. Trên thẻ ghi in thì bác cũng viết in hết là được.
